CHANGELOG.md

```diff
@@ -1,3 +1,119 @@
+### ClickHouse release v21.8.2.19-prestable FIXME!! as compared to v21.7.5.29-stable
+
+#### Backward Incompatible Changes
+
+* Use Map data type for system logs tables (`system.query_log`, `system.query_thread_log`, `system.processes`, `system.OpenTelemetrySpanLog`). These tables will be auto-created with new datatypes; old queries may be incompatible. [#23934](https://github.com/ClickHouse/ClickHouse/pull/23934) ([hexiaoting](https://github.com/hexiaoting)).
```

Comment:
In fact we made it compatible.
Only the queries with SELECT * may suffer but it's alright.

Reply:
I merged this two entries in changelog, and removed from "Backward Incompatible Changes":  [`70db201` (#27271)](https://github.com/ClickHouse/ClickHouse/pull/27271/commits/70db20172ec91e9754e964a39eb980885d589bc4)